clar2wasm/tests/wasm-generation/bitwise.rs

```diff
@@ -0,0 +1,76 @@
+use clar2wasm::tools::crosscheck_compare_only;
```

Comment:
All of the bitwise functions come from the standard, and are already tested with random inputs in *tests/standard/property_tests.rs* (and *unit_tests.rs*).
We also have checks in the package **tests** for contracts using those functions, proving that calling them work.

So I believe that this entire file is not needed, and will just make our testing time even longer.

Reply:
Maybe the tests in **tests/standard** should be crosschecked? That's really the only difference here.

Reply:
@Acaccia @obycode Per our conversation today, should we keep with bitwise property tests or remove them?

I don't think that keeping the tests will hurt since property tests are planned to run out of the core CI actions. What's your take on that? Tks! :-)

Reply:
I think we should leave it unless we end up with a real problem of testing time (after property testing is separated out from unit testing).